### ISSUE TITLE
ERR_REQUIRE_ESM when creating a new feathers project

### ISSUE THREAD

**Issue by patamimbre** [feathersjs/feathers#3392]
### Steps to reproduce

Run `npm create feathers@latest feathers-chat` as described [in the docs](https://feathersjs.com/guides/basics/generator.html)

### Expected behavior

It should create a base project

### Actual behavior

It throws ERR_REQUIRE_ESM error
```
/Users/patamimbre/.npm/_npx/eb274edb682bcb28/node_modules/create-feathers/bin/create-feathers:7
const { Command, commandRunner, chalk } = require('@feathersjs/cli')
                                          ^

Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/patamimbre/.npm/_npx/eb274edb682bcb28/node_modules/@feathersjs/cli/lib/index.js from /Users/patamimbre/.npm/_npx/eb274edb682bcb28/node_modules/create-feathers/bin/create-feathers not supported.
Instead change the require of index.js in /Users/patamimbre/.npm/_npx/eb274edb682bcb28/node_modules/create-feathers/bin/create-feathers to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/Users/patamimbre/.npm/_npx/eb274edb682bcb28/node_modules/create-feathers/bin/create-feathers:7:43) {
  code: 'ERR_REQUIRE_ESM'
}
```

### System configuration

**Module versions** (especially the part that's not working):
create-feathers@5.0.15

**NodeJS version**:
v21.5.0 (tried with node v20 too)

**Operating System**:
Macos Sonoma 14.2.1

**Comment by daffl** [comment]
Whoops, thank you. Should be fixed in 5.0.16

**Comment by MarcGodard** [comment]
@daffl this seems back now when generating a service... Not sure how though.

```
/home/marcgodard/Documents/GitHub/swp-api/node_modules/@feathersjs/cli/lib/index.js:25
const generators_1 = require("@feathersjs/generators");
                     ^

Error [ERR_REQUIRE_ESM]: require() of ES Module /home/marcgodard/Documents/GitHub/swp-api/node_modules/@feathersjs/generators/lib/index.js from /home/marcgodard/Documents/GitHub/swp-api/node_modules/@feathersjs/cli/lib/index.js not supported.
Instead change the require of /home/marcgodard/Documents/GitHub/swp-api/node_modules/@feathersjs/generators/lib/index.js in /home/marcgodard/Documents/GitHub/swp-api/node_modules/@feathersjs/cli/lib/index.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/home/marcgodard/Documents/GitHub/swp-api/node_modules/@feathersjs/cli/lib/index.js:25:22) {
  code: 'ERR_REQUIRE_ESM'
}

Node.js v20.10.0
```
